.rspec

```diff
@@ -1 +1,2 @@
+--format <%= ENV['CI'] ? 'documentation' : 'progress' %>
```

Comment:
Yeah, but it gets a bit tricky because you have to pass options to the underlying framework since we run tests through `parallel_rspec`. I think we would need something like `bin/parallel_rspec --test-options '--format documentation'`, but it gets a bit trickier because the `parallel_rspec` binstub is not a standard one, but does some ARGV handling.

With this the default is still `progress` in development but you can optionally run in CI mode by doing `CI=true bundle exec rake` and get both cucumber and RSpec use a more verbose format.

Reply:
Does this actually work with ERB in the file? I thought it didn't! If so then it's a good solution. 👍🏻

Reply:
I thought so too, but I tried it and it worked!

Reply:
Awesome! Love it. Would be much simpler than doing something else. Good to know. Thanks! 👍🏻